source/dotnet/Library/AdaptiveCards/AdaptiveRichTextBlock.cs

```diff
@@ -60,94 +60,98 @@ public AdaptiveRichTextBlock()
 #endif
         public List<AdaptiveParagraph> Paragraphs { get; set; } = new List<AdaptiveParagraph>();
 
-        [JsonObject(NamingStrategyType = typeof(CamelCaseNamingStrategy))]
-        public class AdaptiveParagraph
-        {
-            [JsonRequired]
-            [JsonConverter(typeof(IgnoreEmptyItemsConverter<AdaptiveTextRun>))]
+
+    }
+
+    [JsonObject(NamingStrategyType = typeof(CamelCaseNamingStrategy))]
+    public class AdaptiveParagraph
```

Comment:
>AdaptiveParagraph [](start = 17, length = 17)

Should this go in it's own file? That's the convention we've been using in UWP.

Reply:
sure thing :)

---
In reply to: [270478025](https://github.com/Microsoft/AdaptiveCards/pull/2581#discussion_r270478025) [](ancestors = 270478025)